src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationService;
+import core.basesyntax.service.operation.OperationServiceImpl;
+import core.basesyntax.service.parse.ParserService;
+import core.basesyntax.service.parse.ParserServiceImpl;
+import core.basesyntax.service.read.ReaderService;
+import core.basesyntax.service.read.ReaderServiceImpl;
+import core.basesyntax.service.report.ReportService;
+import core.basesyntax.service.report.ReportServiceImpl;
+import core.basesyntax.service.strategy.OperationStrategy;
+import core.basesyntax.service.strategy.OperationStrategyImpl;
+import core.basesyntax.service.strategy.handlers.BalanceOperationHandler;
+import core.basesyntax.service.strategy.handlers.OperationHandler;
+import core.basesyntax.service.strategy.handlers.PurchaseOperationHandler;
+import core.basesyntax.service.strategy.handlers.ReturnOperationHandler;
+import core.basesyntax.service.strategy.handlers.SupplyOperationHandler;
+import core.basesyntax.service.write.WriterService;
+import core.basesyntax.service.write.WriterServiceImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        new WriterServiceImpl().write("input.csv","type,fruit,quantity\n"
+                + "b,banana,18\n"
+                + "b,apple,100\n"
+                + "s,banana,14\n"
+                + "p,banana,15\n"
+                + "r,apple,100000\n"
+                + "p,apple,50\n"
+                + "p,banana,5\n"
+                + "s,banana,544");
+        doTask("input.csv", "report.csv");
```

Comment:
let's put files into `src/main/resources` folder

and make constants for file paths